Edit: Server Softwares (#52)
FalixNodes-Software/help-center: _posts/minecraft/general/2021-09-06-server-softwares.md

## Minecraft Java

### PaperMC
[PaperMC](https://papermc.io) is an optimized fork of Spigot that supports Spigot and Bukkit plugins. It also fixes some Vanilla bugs and exploits!

### Purpur
[Purpur](https://purpur.pl3x.net/) is an optimized Fork of Tuinity. It contains extra features and supports Spigot and Bukkit plugins. We recommend using this server software.

### FabricMC
[FabricMC](https://fabricmc.net) is a lightweight Minecraft software used for Fabric mods.

### Vanilla
Vanilla is the default software provided by Minecraft. It does not support plugins or mods.

### Vanilla Snapshot
Vanilla Snapshot is a testing version of Minecraft. Snapshot's usually contain bugs and are not recommended for production use. Just like Vanilla, they do not support plugins or mods.

### Forge
[Forge](https://files.minecraftforge.net/net/minecraftforge/forge) is a Minecraft software that allows players to use Forge mods.

### Cuberite 
[Cuberite](https://cuberite.org/) is an optimized Minecraft server software used for Cuberite plugins.

<br>

## Minecraft Bedrock

> ### Pocketmine(PMMP)

Reviewer: ```suggestion
### Pocketmine (PMMP)
```

> [Pocketmine](https://www.pocketmine.net) is a lightweight Minecraft Bedrock server software. It supports Pocketmine plugins and multiworld system.

Reviewer: > and multiworld system.

What is a "multiworld system"?

Author: You can use many worlds in one server and you can switch from one world to another world using in-game commands.

Reviewer: Is this default with Pocketmine or do you need to install some sort of plugin?
If its default with Pocketmine, any docs/guides you can link?

Author: You will need to install a plugin (Multiworld).

Reviewer: Then please remove this. You already stated Pocketmine supports plugins.
```suggestion
[Pocketmine](https://www.pocketmine.net) is a lightweight Minecraft Bedrock server software that supports Pocketmine plugins.
```

Author: Ok. also "Pocketmine plugins" is correct? Because we are talking about pocketmine in that whole paragraph. Should I Remove "Pocketmine".

Reviewer: I'm fine either way. You can keep the "Pocketmine plugins".

### Altay

> Altay is an optimized fork of Pocketmine(PMMP). It contains extra features and supports Pocketmine plugins.

Reviewer: ```suggestion
[Altay](https://github.com/TuranicTeam/Altay) is an optimized fork of Pocketmine that contains extra features and supports all Pocketmine plugins.
```

### Vanilla Bedrock

> Vanilla Bedrock is the default Bedrock software provided by Minecraft. It supports Behaviour packs but not plugins.

Reviewer: ```suggestion
Vanilla Bedrock is the default Bedrock software provided by Minecraft. It does not support plugins.
```

Behaviour packs are default with Vanilla Bedrock.

## Minecraft Java

### PaperMC
[PaperMC](https://papermc.io) is an optimized fork of Spigot that supports Spigot and Bukkit plugins. It also fixes some Vanilla bugs and exploits!

### Purpur
[Purpur](https://purpur.pl3x.net/) is an optimized Fork of Tuinity. It contains extra features and supports Spigot and Bukkit plugins. We recommend using this server software.

### FabricMC
[FabricMC](https://fabricmc.net) is a lightweight Minecraft software used for Fabric mods.

### Vanilla
Vanilla is the default software provided by Minecraft. It does not support plugins or mods.

### Vanilla Snapshot
Vanilla Snapshot is a testing version of Minecraft. Snapshot's usually contain bugs and are not recommended for production use. Just like Vanilla, they do not support plugins or mods.

### Forge
[Forge](https://files.minecraftforge.net/net/minecraftforge/forge) is a Minecraft software that allows players to use Forge mods.

### Cuberite 
[Cuberite](https://cuberite.org/) is an optimized Minecraft server software used for Cuberite plugins.

<br>

------------------------------------------------------------------------------------

> 

Reviewer: Whats with this?

Author: ![Space](https://user-images.githubusercontent.com/78464152/132289153-e08c2e38-ac84-4634-8197-0aeb67d9de9f.png) this space and line are dividing Java and Bedrock. In my opinion, this is looking good.

Author: ![Withoutspace](https://user-images.githubusercontent.com/78464152/132289711-fb90c2c8-aec9-4cc1-bbe7-745db0a781ca.png)
Without that space and line, it will be going to look like this

Reviewer 2: Imo, remove the line and keep the line break (<br>)

Author: ![Better](https://user-images.githubusercontent.com/78464152/132373509-7dcbc320-0e04-4f1a-bc94-6160dcbbef36.png)
Like this?

## Minecraft Bedrock

### Pocketmine(PMMP)

> [Pocketmine](https://www.pocketmine.net) is a lightweight Minecraft Bedrock server software. It supports Pocketmine plugins and Multi world Systems.

Reviewer: > Multi world Systems.

Why are "multi" and "systems" capitalized?

Author: Because I want to highlight those word but I will going to change "Multi world" to "Multiworld" tomorrow.

Reviewer: Capitalizing a word just to "highlight" is incorrect grammar.
If you really wanted to "highlight" it, just bold it with markdown.

Author: Ok so Which one is better according to you "Multiworld system" or "multiworld system".

Reviewer 3: use `multiworld system`

Reviewer: > Ok so Which one is better according to you "Multiworld system" or "multiworld system".

Whichever one is correct grammar.

### Altay

> Altay is an optimized fork of Pocketmine(PMMP). It contains many extra features and supports all Pocketmine plugins.

Reviewer: ```suggestion
Altay is an optimized fork of Pocketmine(PMMP). It contains extra features and supports Pocketmine plugins.
```

[PaperMC](https://papermc.io) is an optimized fork of Spigot that supports Spigot and Bukkit plugins. It also fixes some Vanilla bugs and exploits!

### Purpur
[Purpur](https://purpur.pl3x.net/) is an optimized fork of Tuinity. It contains extra features and supports Spigot and Bukkit plugins. We recommend using this server software.

### FabricMC
[FabricMC](https://fabricmc.net) is a lightweight Minecraft software used for Fabric mods.

### Vanilla
Vanilla is the default software provided by Minecraft. It does not support plugins or mods.

### Vanilla Snapshot
Vanilla Snapshot is a testing version of Minecraft. Snapshot's usually contain bugs and are not recommended for production use. Just like Vanilla, they do not support plugins or mods.

### Forge
[Forge](https://files.minecraftforge.net/net/minecraftforge/forge) is a Minecraft software that allows players to use Forge mods.

### Cuberite 
[Cuberite](https://cuberite.org/) is an optimized Minecraft server software used for Cuberite plugins.

<br>

## Minecraft Bedrock

### Pocketmine (PMMP)
[Pocketmine](https://www.pocketmine.net) is a lightweight Minecraft Bedrock server software that supports Pocketmine plugins.

### Altay

> [Altay](https://github.com/TuranicTeam/Altay) is an optimized fork of Pocketmine that contains extra features and supports all Pocketmine plugins.

Reviewer: ```suggestion
[Altay](https://github.com/TuranicTeam/Altay) is an optimized fork of Pocketmine that contains extra features and supports Pocketmine plugins.
```

### Vanilla Bedrock
Vanilla Bedrock is the default Bedrock software provided by Minecraft. It does not support plugins.

### Cloudburst

> [Cloudburst](https://cloudburstmc.org/) is an optimized fork of Cloudburst Nukkit written in Java that supports Nukkit plugins.

Reviewer: ```suggestion
[CloudburstMC](https://cloudburstmc.org) (aka NukkitX) is an optimized fork of Nukkit that supports Nukkit plugins.
```
